### DISCUSSION TITLE
CPU Last altes PI Image auf PI 3 gegen neues Linux Image auf PI 5

### DISCUSSION THREAD

**Discussion by chef76** [brutella/hkknx-public#346] [General]
Ich hab beide PIs eben parallel laufen, nur dass der PI 5 nicht an Homekit angebunden ist:
PI 3 mit altem PI OS:
![image](https://github.com/user-attachments/assets/d794de0c-a322-4c55-b141-2812732ed341)
PI 5 mit neuem Linux Image:
![image](https://github.com/user-attachments/assets/71142355-432a-407b-95d3-b7b3e70e24a9)

Dass das Logging der Diagrammdaten mehr Prozessorlast als früher ohne verursacht war mir klar, aber dass kaum ein Unterschied zwischen PI 3 und PI 5 ist wundert mich etwas.

Kann da jemand Erfahrungen beisteuern?

**Comment by brutella** [comment]
Das die CPU Auslastung beim neuen Raspberry Pi image so hoch ist, wundert mich auch. Welche Busauslastung hast du?

**Reply by chef76** [reply]
1-2%, die liefen aber bei den Screenshots parallel und hatten demzufolge auch dieselbe Buslast.
